app/services/testdata/Phase1TestsInvitedStatusGenerator.scala

```diff
@@ -42,22 +43,20 @@ trait Phase1TestsInvitedStatusGenerator extends ConstructiveGenerator {
   def generate(generationId: Int, generatorConfig: GeneratorConfig)(implicit hc: HeaderCarrier) = {
 
     val sjqTest = Phase1Test(
-      cubiksUserId = 117344,
+      cubiksUserId = scala.util.Random.nextInt(10000000),
```

Comment:
At the moment:

- I don't think we test any state transitions (i.e. nothing that would use real cubiks) - we could aim to start doing this eventually which would require proper cubiks user ids
- I don't particularly want the network latency of talking to cubiks in user generation right now

... in this case it doesn't need to be real because we're generating fake test data with the fake id, and using the fake id throughout generation - we never contact actual cubiks for anything. So we get away with it :-)